public/css/widget/check-statistics.less

```diff
@@ -35,6 +35,10 @@
       .bubble {
         position: relative;
       }
+
+      .end .bubble {
+        transform: translate(50%, 0);
```

Comment:
Please put that into progress-bar.less. It's too strongly related to the positioning defined there. Also, maybe couple it with the `placement` class (or rather the new name :wink: )

Reply:
This is necessary here because `.above` is a flex element here (but not in downtime progress bar) with `space-between`. These changes help to move the indicator to its correct position (end of `.above`).